libcxx/test/std/numerics/c.math/hermite.pass.cpp

```diff
@@ -289,6 +296,7 @@ void test() {
     }
   }
 
+#if !(defined(__MVS__) && !defined(__BFP__))
```

Comment:
Could this be solved using 
`if constexpr (std::numeric_limits<Real>::has_infinity)` instead of preprocessor macros?

https://en.cppreference.com/w/cpp/types/numeric_limits/has_infinity